### PR TITLE
Add generate cmd with pp support only and fixed list of typesets

### PR DESCRIPTION
**Pull request by markfuller**
This commit adds the generate CLI command to translate a hard-coded list of typesets to .pp only.  It can translate from go only as it uses the service.Server package directly (rather than the starting all plugins and loading the metadata endpoints as a more evolved solution will eventually do).  The files are generated in the `./plugins/types` folder, and the `./plugins/types` folder is loaded before the `./plugins` folder, so that the types are available to activities defined in `./plugins`.

### CODE REVIEW

File: pkg/loader/loader.go [hunk 1]
```diff
@@ -117,6 +118,9 @@ func (l *Loader) PreLoad(c eval.Context) {
 		// Go plugins
 		l.loadPlugins(c, dir)
 
+		// load typesets from files (NOTE: this is typesets only by convention only at this point, it would attempt to load workflows if found)
```
Comment (scottyw) [inline pkg/loader/loader.go]: I reckon loading from the "types" dir needs to happen before we load any plugins. Notionally, a Go plugin might someday define a workflow that uses types define in some other language.

File: pkg/loader/loader.go [hunk 2]
```diff
@@ -30,6 +30,7 @@ var embeddedPluginNames = []string{
 }
 
 var dir = "./plugins"
+var typesDir = "./plugins/types"
```
Comment (scottyw) [inline pkg/loader/loader.go]: This is a tiny detail, but I wonder should it be "generated" or ".generated" or ".cache"? Something to say that this is a dir of transient things.

File: locales/en_US/default.po [hunk 3]
```diff
@@ -96,6 +96,25 @@ msgstr
...
+#: cmd/lyra/cmd/generate.go:23
+msgid "generateCmdLong"
+msgstr "Generate all typesets in the target language"
+
+#: cmd/lyra/cmd/generate.go:24
+msgid "generateCmdExample" 
+msgstr 
+"\n"
+"  # Generate all typesets in yaml\n"
+"  lyra generate yaml\n"
```
Comment (scottyw) [inline locales/en_US/default.po]: This example wouldn't work at the moment, right?

Comment (markfuller) [reply locales/en_US/default.po]: Agreed, will change.

**Pull request merged**